### PR TITLE
Show UI in not playing states

### PR DESCRIPTION
**Pull request by stonko1994**
## Description
In order to improve UX we decided to show the UI per default in the following states:
- Prepared
- Paused
- Finished

As the UI is visible per default now we decided to lower the default `hideDelay` value.

**This works around multiple issues:**
- First tap not starting playback on iOS 13
- UI hides if user is scrubbing longer than the `hideDelay`

### CODE REVIEW

File: src/ts/components/uicontainer.ts [hunk 1]
```diff
@@ -53,7 +53,12 @@ export class UIContainer extends Container<UIContainerConfig> {
 
     this.config = this.mergeConfig(config, <UIContainerConfig>{
       cssClass: 'ui-uicontainer',
-      hideDelay: 5000,
+      hideDelay: 2000,
+      hidePlayerStateExceptions: [
```
Comment (protyposis) [inline src/ts/components/uicontainer.ts]: This will impact customers' custom UIs so I'd rather go the safe way and do this in our default UI structure only (and schedule the general change for v4).

File: src/ts/components/uicontainer.ts [hunk 2]
```diff
@@ -236,8 +243,6 @@ export class UIContainer extends Container<UIContainerConfig> {
     uimanager.getConfig().events.onUpdated.subscribe(() => {
       updateState(PlayerUtils.getState(player));
     });
-    // Init in current player state
-    updateState(PlayerUtils.getState(player));
```
Comment (protyposis) [inline src/ts/components/uicontainer.ts]: Why did you remove the initialization?

File: src/ts/components/uicontainer.ts (hunk 1 above)
Comment (stonko1994) [reply src/ts/components/uicontainer.ts]: > ... in our default UI structure ...

Do you mean only within `UIFactory`?

I thought about the same but decided against as we want to change the default behaviour in all UI variants. But thinking again it would make more sense to not change components behaviour.

File: src/ts/components/uicontainer.ts (hunk 2 above)
Comment (stonko1994) [reply src/ts/components/uicontainer.ts]: The problem here is that the `updateState` now actually triggers a `showUI` before other components had a chance to subscribe to the `onControlsShow` callback. This happens if we are in a `hidingPreventedState` (Prepared). But within the `onShow` we have a flag (`isUiShown`) which will change into `true` but nothing is actual visible.

Call stack:
- `updateState`
- `stateChanged.dispatch`
- Calls `stateChanged.subscribe` callback in `uicontainer`
- `hidingPrevented` returns `true`
- calls `showUI`
- switches the `isUiShown` flag
- a second call would not trigger the `showUI` again

---

I think it's safe to remove the initialization here because the `uimanager` will fire `onUpdated` after initialization (if player is ready). Or the player will fire an appropriated event when it's ready.

Comment (protyposis) [reply src/ts/components/uicontainer.ts]: Makes sense 👍

File: src/ts/components/uicontainer.ts (hunk 1 above)
Comment (protyposis) [reply src/ts/components/uicontainer.ts]: Yes that's what I meant :)

Comment (stonko1994) [reply src/ts/components/uicontainer.ts]: But thinking about it a third time maybe it's better to have it here 🤔 
My argument for this is that the value in `UIFactory` would override this one here. So if a customer changed it already within the `UIContainer` he would need to remove it again from the `UIFactory` (all variants). 

On the other side if a customer doesn't want this change he just needs to change it here once 🤔 

Maybe both variants have advantages and disadvantages. Not sure what's the better place to do this 'breaking' change.
